src/free-style.spec.ts

```diff
@@ -1,5 +1,5 @@
-import test = require('blue-tape')
-import crypto = require('crypto')
+import * as test from 'blue-tape'
+import * as crypto from 'crypto'
```

Comment:
Typescript *does* handle this, it is however non-valid ES6 though. Typescript, `tsc` to be more exact, actually suggests using it as such when attempting to build to ES2015 modules and there are `import foo = require()` style imports.